### DISCUSSION TITLE
Github student reapplication problem

### DISCUSSION THREAD

**Discussion by Oxnas32** [community/community#146769] [GitHub Education]
### Hi everyone,

I tried to apply for the Github Student Develer Pack but i was rejected as I did not confugre properly the billing information or the two factor authentication. I tried to reapply but the only option to submit proof that the page gives me now is via my laptop camera, which is a problem since I have no easy access to the physical form of my university enrollment letter (the proof of academic status i was submitting). Also, It requires me to add proof of the reason why I'm not on campus, which makes no sense since I'm, in fact, on campus. What should I do?

**Comment by Juan-Carlos-D** [comment]
Unfortunately I'm also in the same situation as you. I followed everything in this FAQ page, but no luck so far:
https://github.com/orgs/community/discussions/111352

I hope you get it resolved!

**Comment by davevad93** [comment]
Hi @Oxnas32 ,

About your first problem, you may submit documents other than your ID or certificate of enrollment, as indicated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student):

 - **Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.**

Regarding the second one, it could be that GitHub has incorrect information about your school, try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
